### ISSUE TITLE
[Bug Report][3.5.1] v-tabs flip out when on v3.5.1 or above...but only in production

### ISSUE THREAD

**Issue by Fedorov1977** [vuetifyjs/vuetify#19219]
### Environment
**Vuetify Version:** 3.5.1
**Vue Version:** 3.4.17
**Browsers:** Chrome 121.0.0.0
**OS:** Windows 10

### Steps to reproduce
I'm sorry but I don't know how to reproduce. Let me describe the problem.
I use v-tabs and have 8 tabs inside. I noticed that when clicking through the tabs in production env (deployed with run build from Azure) they flip out after switching between three tabs.
It's always after three tabs regardless of which three ones I click and only in those environments using a production build.

What seems to be happening is that I can see "fragments" of the vue-component in the tab "swooping by" but it looks like it either doesn't "stop" inside the tab and sometimes it looks like it is "hiding" behind the v-card that surrounds the components inside because I can sort of see the borders of the vue-component "slightly sticking out" around the left and right edges of the window-item. There are no error messages in Chrome developer console.

After trying downgrading vuetify I've now concluded that this happens on version 3.5.1 (or above) of vuetify. 
I've also tried vue v3.4.15 and v3.4.17 but it made no difference, it still works as long as used with vuetify 3.4.0. I have not tried a lower version of vuetify than 3.4.0 because I did not want to downgrade more than necessary.

### Expected Behavior
Tabs display correct content when clicking on different tabs

### Actual Behavior
After 3 clicks the content does not display correctly. It seems to be after 3 clicks regardless of which tabs I click first. The first 3 tabs I click display their content correctly, after that it does not.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqLrq7VUTLWM9EzNFcCMUz1jJViATjmBLM=)

### Other comments
As a developer I realize the "ungratefullness" of getting a bug report that is not easy to reproduce...

**Comment by peteralfredoflynn** [comment]
I had a similar issue using vuetify 3.5.3 and Vue 3.4.18. Upgrading to Vue 3.4.19 fixed it for me.

**Comment by Fedorov1977** [comment]
@peteralfredoflynn I think you're definitely on to something! I tried now upgrading both vue and vuetify to latest version (vue 3.4.19 and vuetify 3.5.4). It seems to work fine again with a production build. I had not tried that combo of versions before. It seems that vue 3.4.19 is released very recently so I had only tried versions 3.4.15 up to, and including, 3.4.18 of vue.

So to summarize:
It seems that before vue 3.4.19 there was some bug potentially with the v-tabs when used with vuetify versions 3.51 or above.
